modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js

```diff
@@ -8,40 +8,65 @@
  * permissions and limitations under the License, including but not limited to
  * distribution rights of the Software.
  */
-
 import {useQuery} from '@apollo/client';
 import ClayForm from '@clayui/form';
 import {FieldArray, Formik} from 'formik';
-import {useEffect, useMemo} from 'react';
-import {getDXPCloudPageInfo} from '../../../../common/services/liferay/graphql/queries';
+import {useEffect, useMemo, useState} from 'react';
+import client from '../../../../apolloClient';
+import {
+	addAnalyticsCloudWorkspace,
+	addIncidentReportAnalyticsCloud,
+	getAnalyticsCloudPageInfo,
+} from '../../../../common/services/liferay/graphql/queries';
 import {
 	isLowercaseAndNumbers,
+	isValidEmail,
+	isValidEmailDomain,
 	isValidFriendlyURL,
 	maxLength,
 } from '../../../../common/utils/validations.form';
 import {Button, Input, Select} from '../../../components';
+import useBannedDomains from '../../../hooks/useBannedDomains';
 import getInitialAnalyticsInvite from '../../../utils/getInitialAnalyticsInvite';
 import Layout from '../Layout';
+
 import IncidentReportInput from './IncidentReportInput';
 
-const INITIAL_SETUP_ADMIN_COUNT = 1;
+const MAX_LENGTH = 255;
 
 const SetupAnalyticsCloudPage = ({
+	errors,
 	handlePage,
 	leftButton,
+	onClose,
 	project,
 	setFieldValue,
+	touched,
 	values,
 }) => {
-	const {data} = useQuery(getDXPCloudPageInfo, {
+	const [baseButtonDisabled, setBaseButtonDisabled] = useState(true);
+
+	const bannedDomainsOwnerEmail = useBannedDomains(
+		values?.activations?.ownerEmailAddress,
+		500
```

Comment:
I think it will be better if declare a constant for this value (500) , what do you think?

Reply:
Good, thank you!